### DISCUSSION TITLE
PowerFlow Display is suddenly empty

### DISCUSSION THREAD

**Discussion by marcbaier** [jasonacox/Powerwall-Dashboard#260] [Q&A]
My Powerwall Dashboard has been working fine for months and suddenly the PowerFlow animation on the right is just showing a white frame. Does anyone know what could cause this and how it can be fixed ?

I don't see to be getting an error message either. Thanks so much for any help !
<img width="421" alt="Screen Shot 2023-04-27 at 15 48 36 PM" src="https://user-images.githubusercontent.com/97667985/234882621-3d2597d7-9870-4a23-9c6e-e4baaf69ebb5.png">

**Comment by ThePnuts** [comment]
Couple of observations I have seen, but might not be your issue.

The white box I get in Chrome, other browsers like Safari show it is transparent, just Chrome does white like that for me.

As for the graphic missing. I can only get the graphic on 1 dashboard at a time. Like if I have it open on my desktop, then open it on the laptop, its just empty space. Or even a 2nd tab\window after the first, I get the white box like that.

If you log directly into the powerwall do you still see the graphic?

**Comment by jasonacox** [comment]
Hi @marcbaier - 

This will happen if you load the power flow animation in more than one tab, or different windows of the same browser. Check to make sure you don't have more than one copy of the dashboard open. This seems to happen with Chrome and Brave but doesn't happen with Safari, Edge or Firefox, at least for me.  Anyway, see if that fixes it.  If not:

See if this is a pypowerwall issue, go to: http://localhost:8675/example.html (replace localhost with the address of the dashboard host).  It should load the example iFrame render:

<img width="621" alt="image" src="https://user-images.githubusercontent.com/836718/235040547-571dd249-2a58-47e2-a4b5-90d9f3ff8c2f.png">

**Reply by marcbaier** [reply]
Thanks so much for the quick reply. I honestly don't know what the issue was as it now magically is working again. I might indeed have had another tab open somewhere in the background.
